modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -8,6 +8,8 @@ var _ = require('lodash'),
   path = require('path'),
   errorHandler = require(path.resolve('./modules/core/server/controllers/errors.server.controller')),
   mongoose = require('mongoose'),
+  multer = require('multer'),
+  defaultEnvConfig = require(path.resolve('./config/env/default')),
```

Comment:
I think you could just use the global config here..
`config = require(path.resolve('./config/config'))`

A user might add this setting to one of their env configs (i.e. Production) to override the default setting. WDYT?